src/promptflow-tracing/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import json
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def dev_connections() -> dict:
+    with open(
+        file=(Path(__file__).parent.parent / "connections.json").resolve().absolute().as_posix(),
```

Comment:
connection needed?

Reply:
yes, we have test covering the openai injector logic, which requires connection - i use live here for now